core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -536,6 +538,45 @@ public void testBasicCreateTable() {
     assertThat(table.properties()).as("Should have table properties").isNotNull();
   }
 
+  @Test
+  public void testCreateGeospatialTable() {
```

Comment:
I think I commented elsewhere, but this is not a good place for an end-to-end test creating a table with a geospatial column. These tests assert the behavior of catalogs.

Reply:
Moved it to `TestGeospatialTable` and switched from `HadoopCatalog` to `InMemoryCatalog`.